### ISSUE TITLE
[Question] PHP 8.1

### ISSUE THREAD

**Issue by lenamtl** [spipu/html2pdf#732]
Hi, does the actual release is compatible with PHP 8.1?

**Comment by jausions** [comment]
Apparently not completely, I will see if I can do a PR.

```
A PHP Error was encountered

Severity:    8192
Message:     Implicit conversion from float 208.87199059649922 to int loses precision
Filename:    vendor\spipu\html2pdf\src\MyPdf.php
Line Number: 1436

Backtrace:
	File: vendor\spipu\html2pdf\src\MyPdf.php
	Line: 1436
	Function: str_repeat

	File: vendor\spipu\html2pdf\src\Html2Pdf.php
	Line: 504
	Function: createIndex
```

**Comment by jausions** [comment]
And it is worth mentioning that TCPDF is not fully compatible with PHP 8.1 yet, so html2pdf can't be either.

**Comment by vespino** [comment]
Working fine so far.

**Comment by spipu** [comment]
fixed in the last version 5.2.6
